### PR TITLE
add several offers methods

### PR DESCRIPTION
**Pull request by moofkit**
- add `Offer::create` method
- add `Offer::addTargetCountry`method
- add `Offer::unblockAffiliate` method

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,9 @@ script: bundle exec rspec spec
 before_install:
   - gem install bundler
 rvm:
-  - 2.2.5
-  - 2.3.1
+  - 1.9.3
+  - 2.0.0
+  - 2.1.8
+  - 2.2.4
+  - 2.3.0
```
Comment (miry) [inline .travis.yml]: I think it is yours commit that updated those. https://github.com/applift/hasoffersv3/commit/48f2adcb9c8b484f9077a13f35520ab1752665ed

Pls rebase against the master.

File: lib/hasoffersv3/affiliate.rb [hunk 2]
```diff
@@ -9,11 +9,6 @@ def find_by_id(params = {})
       get_request 'findById', params
     end
 
-    def update(params = {})
-      requires! params, [:id, :data]
-      post_request 'update', params
-    end
-
```
Comment (miry) [inline lib/hasoffersv3/affiliate.rb]: Pls rebase against master

**Comment by miry** [comment]
@moofkit pls rebase against master.

**Comment by moofkit** [comment]
sorry, didn't notice before pushing

**Comment by miry** [comment]
@moofkit Thanks 👍

**Pull request merged**